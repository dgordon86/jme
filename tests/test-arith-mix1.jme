main() {
a = 2;
b = 17.6;
print(a + b);
print(a - b);
print(a * b);
print(b / a);
}
