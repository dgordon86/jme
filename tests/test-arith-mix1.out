19.6
-15.6
35.2
8.8
